index.html

```diff
@@ -1687,7 +1717,7 @@ <h1>Download</h1>
 									<br>We provide images based on the older Debian Bullseye release as well, as not all software in our catalogue is compatible with Debian Bookworm yet. All info about this can be found <a href="https://dietpi.com/blog/?p=3128" target="_blank" rel="noopener">here</a>.
 								</div>
 							</div>
-							<p>The new Orange Pi flagship SBC, featuring the new Rockchip RK3588 SoC: A quad-core 4x Cortex-A76 @2.4 GHz + 4x Cortex A55 @1.8 GHz is making this the by far fastest SoC ever tested with DietPi.</p>
+							<p>The new Orange Pi flagship SBC, featuring the new Rockchip RK3588 SoC: A quad-core 4x Cortex-A76 @2.4 GHz + 4x Cortex A55 @1.8 GHz is making this the by far fastest SoC ever tested with DietPi. The 5B variant, which is supported by this image as well, additionally has an onboard WiFi 6 and Bluetooth adapter, eMMC flash, but no M.2 socket.</p>
```

Comment:
<p>The new Orange Pi flagship SBC, featuring the new Rockchip RK3588 SoC: A quad-core 4x Cortex-A76 @2.4 GHz + 4x Cortex A55 @1.8 GHz is making this (as well as the NanoPi M6 / NanoPC T6) the by far fastest SoCs ever tested with DietPi. The 5B variant, which is supported by this image as well, additionally has an onboard WiFi 6 and Bluetooth adapter, eMMC flash, but no M.2 socket.</p>

Reply:
They all share the same SoC, so the original sentence is correct. I think it is fine to leave it like that, as otherwise we'd consequently need to update it with every new RK3588 SoC SBC. Right now, the sentence is present on each vendor's first RK3588 SBC, and every new variant contains some diff info.